tests/modules/sightings/resources/test_create_sighting.py

```diff
@@ -82,9 +82,57 @@ def test_create_and_delete_sighting(db, flask_app_client, researcher_1, staff_us
     sighting = Sighting.query.get(sighting_id)
     assert sighting is not None
 
+    response = sighting_utils.read_sighting(
+        flask_app_client, researcher_1, sighting_id, expected_status_code=200
+    )
+    assert response.json['id'] == sighting_id
+
     # upon success (yay) we clean up our mess
     sighting_utils.cleanup_tus_dir(transaction_id)
     sighting_utils.delete_sighting(flask_app_client, researcher_1, sighting_id)
 
     post_ct = test_utils.multi_count(db, (Sighting, Encounter, Asset, Submission))
     assert orig_ct == post_ct
+
+
+def test_create_anon_and_delete_sighting(db, flask_app_client, staff_user):
+    return  # DISABLING TEST FOR NOW
```

Comment:
crafty.
The other way to do this is to change the function name, if it doesn't start with test_ pytest doesn't run it

Reply:
i have wondered about that!   thx.